Source/Core/DolphinQt/Settings.cpp

```diff
@@ -801,6 +801,62 @@ void Settings::SetWiiSpeakMuted(bool muted)
   emit WiiSpeakMuteChanged(muted);
 }
 
+bool Settings::IsLogitechMic1Muted() const
```

Comment:
Instead of having four copies of basically the same function, please make the function just take an index value parameter.
If you need help with the `connect` calls you can ask in discord.